src/signalingclient/SignalingClient.ts

```diff
@@ -64,6 +65,22 @@ export default interface SignalingClient {
    */
   subscribe(settings: SignalingClientSubscribe): void;
 
+  /**
+   * Sends a remote video update frame with the given configurations and removed streams.  These can be
+   * sent in parallel to `subscribe` without issue, but note that `subscribe` will overwrite any configuration
+   * set here.
+   *
+   * Note that the `addedOrUpdated` is basically a flattened map with `SignalingClientVideoSubscriptionConfiguration.mid`
+   * being the key, which is why we only need to pass the mid when removing.
```

Comment:
You win today's award for good docs.
![otter](https://user-images.githubusercontent.com/40039049/134032141-3f5e0a37-bbf9-489f-b7e9-a1ed820de82a.jpg)